### DISCUSSION TITLE
[gatsby-source-shopify] Status of querying Shopify Pages

### DISCUSSION THREAD

**Discussion by AlexanderProd** [gatsbyjs/gatsby#35360] [Ideas / Feature Requests]
Hi, 

I wanted to ask what the status of Shopify Pages in `gatsby-source-shopify` is? 
If I remember correctly it was possible in an older version of `gatsby-source-shopify` to query Shopify pages but this feature seems to have been removed. 

Are there plans on adding it back? The Shopify pages are easily accessible through the Shopify Admin API and even the Storefront API.

**Comment by jackonawalk** [comment]
Hi @AlexanderProd, we don't have plans to support Shopify content at this time. The new source plugin uses the Admin API's Bulk Operation which does not support content entries. If Shopify were to add support, we would love to bring this to the source plugin. A potential work around would be to use [gatsby-graphql-toolkit](https://github.com/gatsbyjs/gatsby-graphql-toolkit) to build a custom source plugin for content entries.

**Reply by DennisKraaijeveld** [reply]
> Hi @AlexanderProd, we don't have plans to support Shopify content at this time. The new source plugin uses the Admin API's Bulk Operation which does not support content entries. If Shopify were to add support, we would love to bring this to the source plugin. A potential work around would be to use [gatsby-graphql-toolkit](https://github.com/gatsbyjs/gatsby-graphql-toolkit) to build a custom source plugin for content entries.

And what about translations? Great plugin and framework, but it is lacking translations and this is a big let down for bigger companies/ecommerce stores
